src/Jackett.Common/Definitions/riperam.yml

```diff
@@ -894,6 +901,11 @@ search:
     st: 0
     ch: 300
     t: 0
+  keywordsfilters:
+    - name: re_replace # S01 to 1
+      args: ["(?i)\\bS0*(\\d+)\\b", "$1"]
+    - name: re_replace # S01E01 to 1 1
+      args: ["(?i)\\bS0*(\\d+)E0*(\\d+)\\b", "$1 $2"]
```

Comment:
RiperAM ignores keywords which are fewer than 3 characters, so season/episode numbers on their own would just be ignored, and only the series name would be searched for. You could add an `andmatch` filter to the row selector and that should semi-resolve the issue.

Reply:
This is more just a quick hack to allow series search from sonarr.

Reply:
Yea, that's fine, but for those searching manually, the `andmatch` filter will be useful.